app/controllers/v0/profile/transactions_controller.rb

```diff
@@ -11,14 +11,17 @@ def status
           service,
           transaction_params[:transaction_id]
         )
+
         raise Common::Exceptions::RecordNotFound, transaction unless transaction
+        Vet360Redis::Cache.invalidate(@current_user)
+
         render json: transaction, serializer: AsyncTransaction::BaseSerializer
       end
 
       def statuses
         transactions = AsyncTransaction::Vet360::Base.refresh_transaction_statuses(@current_user, service)
 
-        Vet360Redis::Cache.invalidate(@current_user) if transactions.present?
+        Vet360Redis::Cache.invalidate(@current_user)
```

Comment:
Same question as above. Do we want to automatically invalidate the cache every time the endpoint is hit?